README.md

```diff
@@ -178,8 +232,12 @@ Rouge uses [Semantic Versioning 2.0.0][sv2].
 
 ## Tips
 
-I don't get paid to maintain rouge. If you've found this software useful, consider dropping a tip in the [bucket](http://cash.me/$jneen).
+I don't get paid to maintain rouge. If you've found this software useful,
```

Comment:
```suggestion
I don't get paid to maintain Rouge. If you've found this software useful,
```

Reply:
Nobody has put anything in here in years, if ever lol. I say we remove it.

Reply:
Maybe replace with something like "Rouge is maintained mostly by unpaid volunteers"

Reply:
I'll redraft this.